crates/primitives/src/key.rs

```diff
@@ -236,10 +236,12 @@ const _: () = {
     };
 
     impl TypeInfo for Key {
+        type Identity = Self;
+
         fn type_info() -> Type {
             Type::builder()
                 .path(Path::new("Key", "ink_primitives"))
-                .composite(Fields::unnamed().field_of::<[u8; 32]>())
+                .composite(Fields::unnamed().field_of::<[u8; 32]>("[u64; 4]"))
```

Comment:
the `[u64; 4]` is just an optimization but we want to serialize it as `[u8; 32]` ideally to avoid problems with endianess.

Reply:
Hm yeah it is still a `[u8; 32]`, I was just mimicking what the derive would put there for the "display name". However the derive would also have generated `field_of::<[u64; 4]>`, which I guess is the point of having this hand-rolled impl. I will change it.